src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_xBind.cs

```diff
@@ -39,4 +43,65 @@ public async Task When_xBind_With_Cast_Default_Namespace()
 
 		Assert.AreEqual("Hello", SUT.tb.Text);
 	}
+
+	[TestMethod]
+	public async Task When_XBind_TargetDisposed_Test()
+	{
+		FeatureConfiguration.Asd.EnableFix = true;
+
+		// load the view with the x:Bind
+		var SUT = new When_XBind_TargetDisposed();
+		var vm = SUT.ViewModel;
+		var wrSUT = new WeakReference(SUT);
+		await UITestHelper.Load(SUT);
+
+		// make sure it is disposed
+		SUT = null; // we must not keep any hard-reference to it
+		await UITestHelper.Load(new Border { Width = 50, Height = 50 });
+		if (!await WaitUntilCollected())
+		{
+			Assert.Fail("Timed out on waiting the view to be disposed");
+		}
+
+		try
+		{
+			// trigger an INotifyCollectionChanged update via ObservableCollection
+			vm.Items.Clear();
+		}
+		catch (Exception e)
```

Comment:
Not sure if we need to `catch`. If an exception is thrown, the test should fail anyways, and the failure will have the stack trace, while `Assert.Fail(e.Message)` is losing this info.